### PR TITLE
[GH-5] feature(storybook): add storybook

### PR DESCRIPTION
**Pull request by kcubero27**
- [ ] add URL to .env
- [ ] add .env and .env.dist

### CODE REVIEW

File: .storybook/addon.js [hunk 1]
```diff
@@ -0,0 +1,4 @@
+import "@storybook/addon-options/register";
+import "storybook-addon-jsx/register";
```
Comment (jesugmz) [inline .storybook/addon.js:2]: What's the difference between prepending an @? cannot find in the Storybook docs or JS general doc :thinking:

File: .storybook/config.js [hunk 2]
```diff
@@ -0,0 +1,28 @@
...
+import "./index.css";
+
+addDecorator(withA11y);
+
+addParameters({
+  options: {
+    theme: create({
+      base: "light",
+      brandTitle: "NextJS",
+      // brandUrl: 'https://example.com',
```
Comment (jesugmz) [inline .storybook/config.js:14]: So far it's more than fine but as future improvement this values maybe good to be moved into a .env file.

File: .storybook/addon.js (hunk 1 above)
Comment (jesugmz) [inline .storybook/addon.js:2]: What's the difference between prepending an @? cannot find in the Storybook docs or JS general doc :thinking:

File: .storybook/config.js (hunk 2 above)
Comment (jesugmz) [inline .storybook/config.js:14]: So far it's more than fine but as future improvement this kind of values maybe good to be moved into a .env file.

File: .storybook/addon.js (hunk 1 above)
Comment (kcubero27) [reply .storybook/addon.js:2]: Basically it's the organization. For example, we could create a lot of libraries under the name of @pepecos/mylibrary1, @pepecos/library2 etc
https://docs.npmjs.com/about-scopes
But they are still libraries, it's only that they put the company or team, organization as a "prefix".

File: .storybook/config.js (hunk 2 above)
Comment (kcubero27) [reply .storybook/config.js:14]: I'll do that 👍

Comment (kcubero27) [reply .storybook/config.js:14]: - [ ] Add variable to .env and .envdist

File: .storybook/addon.js (hunk 1 above)
Comment (jesugmz) [reply .storybook/addon.js:2]: Oh, thanks for the info!

I see great advantage from the doc:

> Unscoped packages are always public.
> Private packages are always scoped.
> Scoped packages are private by default; you must pass a command-line flag when publishing to make them public.

**Comment by kcubero27** [comment]
@jesugmz I will merge this task. I realised there is a mistake with the config of postcss. I'll tackle it in another task.

**Pull request merged**